jsonnet/kube-prometheus/kube-prometheus-custom-metrics.libsonnet

```diff
@@ -14,19 +12,12 @@ local k = import 'github.com/ksonnet/ksonnet-lib/ksonnet.beta.4/k.libsonnet';
             seriesFilters: [],
             resources: {
               overrides: {
-                namespace: {
-                  resource: 'namespace'
-                },
-                pod: {
-                  resource: 'pod'
-                }
+                namespace: { resource: 'namespace' },
+                pod: { resource: 'pod' },
               },
             },
-            name: {
-              matches: '^container_(.*)_seconds_total$',
-              as: ""
-            },
-            metricsQuery: 'sum(rate(<<.Series>>{<<.LabelMatchers>>,container!="POD"}[1m])) by (<<.GroupBy>>)'
+            name: { matches: '^container_(.*)_seconds_total$', as: '' },
+            metricsQuery: 'sum(rate(<<.Series>>{<<.LabelMatchers>>,container!="POD"}[1m])) by (<<.GroupBy>>)',
```

Comment:
I curious is inlining in this file an effect of jsonnetfmt or was this manual?

Reply:
I wish this would be automated. I manually changed those.